src/dotnet/commands/dotnet-update/tool/LocalizableStrings.resx

```diff
@@ -0,0 +1,183 @@
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="PackageIdArgumentName" xml:space="preserve">
+    <value>PACKAGE_ID</value>
+  </data>
+  <data name="PackageIdArgumentDescription" xml:space="preserve">
+    <value>NuGet Package Id of the tool to update.</value>
+  </data>
+  <data name="SpecifyExactlyOnePackageId" xml:space="preserve">
+    <value>Please specify one tool Package Id to update.</value>
+  </data>
+  <data name="VersionOptionDescription" xml:space="preserve">
+    <value>Version of the tool package in NuGet.</value>
+  </data>
+  <data name="SourceOptionDescription" xml:space="preserve">
+    <value>Specifies a NuGet package source to use during update.</value>
+  </data>
+  <data name="SourceOptionName" xml:space="preserve">
+    <value>SOURCE</value>
+  </data>
+  <data name="CommandDescription" xml:space="preserve">
+    <value>Updates a tool for use on the command line.</value>
+  </data>
+  <data name="ConfigFileOptionDescription" xml:space="preserve">
+    <value>The NuGet configuration file to use.</value>
+  </data>
+  <data name="FrameworkOptionDescription" xml:space="preserve">
+    <value>The target framework to update the tool for.</value>
+  </data>
+  <data name="NuGetConfigurationFileDoesNotExist" xml:space="preserve">
+    <value>NuGet configuration file '{0}' does not exist.</value>
+  </data>
+  <data name="UpdateSucceeded" xml:space="preserve">
+    <value>Tool '{0}' (from version '{1}' to version'{2}') was successfully updated.</value>
+  </data>
+  <data name="GlobalOptionDescription" xml:space="preserve">
+    <value>Update user wide.</value>
+  </data>
+  <data name="UpdateFullCommandNameLocalized" xml:space="preserve">
+    <value>.NET update Command</value>
+  </data>
+  <data name="ToolAlreadyInstalled" xml:space="preserve">
+    <value>Tool '{0}' is already installed.</value>
+  </data>
+  <data name="UpdateToolCommandNeedGlobalOrToolPath" xml:space="preserve">
+    <value>Please specify either the global option (--global) or the tool path option (--tool-path).</value>
+  </data>
+  <data name="UpdateToolCommandInvalidGlobalAndToolPath" xml:space="preserve">
+    <value>(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</value>
+  </data>
+  <data name="ToolPathDescription" xml:space="preserve">
+    <value>Location of shim to access tool</value>
```

Comment:
Should we use the string we're going to change the install/uninstall command descriptions to (i.e. the one we had the meeting with Kathleen about)?  Thus we won't forget to update this location as well.

Reply:
I forget. I will correct it

Reply:
Updated to what we deicded